### PR TITLE
fix: remove mov file support in create drop

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
Issue - https://linear.app/showtime/issue/SHOW2-1317/remove-mov-type-because-opensea-doesnt-support-it
- Remove .mov upload support from frontend

# How?
- Show error instead of using input `accept` prop. This is tricky with cross platform as native image picker doesn't seem to have a similar prop.

# Test Plan
- Try uploading .mov file on drop form

### CODE REVIEW

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

**Review by alantoa — APPROVED** [review]
LGTM ✨

File: packages/app/components/drop/drop-form.tsx [hunk 1]
```diff
@@ -281,6 +282,25 @@ export const DropForm = () => {
     );
   }
 
+  const handleFileChange = (file: string | File) => {
+    let extension;
+    // On Native file is a string uri
+    if (typeof file === "string") {
+      extension = file.split(".").pop();
```
Comment (alantoa) [inline packages/app/components/drop/drop-form.tsx:289]: if using MIME types to get it will be better because some files haven't the extension.

Comment (intergalacticspacehighway) [reply packages/app/components/drop/drop-form.tsx:289]: yeah, we use MIME type on web. On native, I don't think there's a way yet. https://docs.expo.dev/versions/latest/sdk/imagepicker/#imageinfo

**Pull request merged**